pileup_CTCF.ipynb

```diff
@@ -0,0 +1,649 @@
+{
```

Comment:
<p>Ha I am not used at looking at just observed pileups. Maybe vmin should be higher?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Yeah, I also do not use it. Yet it seems to be important for the demonstration of functionality. How about now?